code/modules/projectiles/projectile/energy/net_snare.dm

```diff
@@ -31,24 +39,21 @@
 
 /obj/effect/nettingportal/Initialize()
 	. = ..()
-	var/obj/item/device/beacon/teletarget = null
-	for(var/obj/machinery/computer/teleporter/com in GLOB.machines)
-		if(com.target)
-			if(com.power_station && com.power_station.teleporter_hub && com.power_station.engaged)
-				teletarget = com.target
 
-	addtimer(CALLBACK(src, .proc/pop, teletarget), 30)
+	addtimer(CALLBACK(src, .proc/pop), 30)
 
-/obj/effect/nettingportal/proc/pop(teletarget)
+/obj/effect/nettingportal/proc/pop()
 	if(teletarget)
```

Comment:
🚲 🏠 but `&& loc != TT` (you'll have to move the TT up)

If you get netted where the beacon is you'll just get thrown somewhere nearby instead of nothing happening.